hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/SnapshotContext.java

```diff
@@ -143,7 +166,8 @@ synchronized void initTaskletCount(int totalTaskletCount, int highPriorityTaskle
      * {@code SnapshotOperation} and send barriers to such processor before
      * the {@code SnapshotOperation} is executed on this member.
      */
-    synchronized CompletableFuture<SnapshotOperationResult> startNewSnapshot(long snapshotId, boolean isTerminal) {
+    synchronized CompletableFuture<SnapshotOperationResult> startNewSnapshot(
+            long snapshotId, int dataMapIndex, boolean isTerminal) {
```

Comment:
since snapshot can't start anymore just because of barrier from remote member we should remove the comment and also simplify the related logic

Reply:
fixed